ORStools/gui/ORStoolsDialog.py

```diff
@@ -284,7 +285,10 @@ def run_gui_control(self) -> None:
             self.dlg.line_tool = maptools.LineTool(self.dlg)
 
         except exceptions.ApiError as e:
-            if "Could not find routable point within a radius of 350.0 meters of specified coordinate":
+            parsed = json.loads(e.message)
+            error_code = int(parsed["error"]["code"])
+            if error_code == 2010:
+                # Error thrown by ORStools/common/client.py, line 243, in _check_status
```

Comment:
I'd rather this be a comment directly on the `try` or on the `except`-statement.